src/transition.js

```diff
@@ -76,13 +76,13 @@ export function withCurve(transition, curve) {
  * a specified as a map from CSS property names to transition functions for
  * each of these properties.
  * @param {!Element} element
- * @param {!Object<string, !TransitionDef>} styles
- * @return {!TransitionDef<void>}
+ * @param {!Object<string, !TimeTransitionDef>} styles
+ * @return {!TimeTransitionDef<void>}
```

Comment:
Ok. If you can tell them apart then I think this change is fine. Let's add a FixIt item to get rid of one of them though.